samples/snippets/src/test/java/com/example/errorreporting/QuickStartIT.java

```diff
@@ -27,6 +27,6 @@
   @Test
   public void testQuickStart() throws Exception {
     // Ensure quick start runs without any exception
-    QuickStart.main(new String[]{});
+    QuickStart.main(new String[] {});
```

Comment:
Is it really necessary?

Reply:
putting this back

Reply:
Looks like owlbot is putting it back to fix formatting